Class-10-(Scope-Exceptions-Types).md

```diff
@@ -0,0 +1,23 @@
+## In the prep work this week, the students learned:
+1. More about scope. Scope was covered briefly in the Functions chapter, however, now students have covered it more in-depth in terms of the order code executes in.
+1. What exceptions are.
+1. How to raise an exception.
+1. More about what a primitive data type is.
+
+## Announcements
+1. More info to come... when in doubt, check with your class Candidate Engagement Manager!
+
+## Large Group Time (Instructor)
+
+### Class 10 Topics That Require Careful Attention
+1. Conversation starters:
+   1. What is the difference between different types of errors.
+   1. What are some more common errors?
+   1. How do we raise an exception?
+   1. How do we handle exceptions?
+
```

Comment:
Other topics:
a) What does "variable hoisting" mean, and what does it look like?
b) Why would we want to throw an exception?
c) Why should we try to avoid variable shadowing?
d) Q & A for the exercises.
e) Studio intro: Non-coding activity this week on Strategic Debugging.

Reply:
Done! Thanks!